### DISCUSSION TITLE
請問有沒有sector-checker新版本?

### DISCUSSION THREAD

**Discussion by seantw1974** [filecoin-project/lotus#8249] [General Q&A] [answered]
我下載一年多年版本...
用 go build 都不會過..
依賴都不一樣了...

**Comment by weizihua** [comment]
改一下就好了

**Comment by seantw1974** [comment]
更新一下:
1. 試著改了一下,有幾個依赖過不了
最後放在lotus 源碼底下 /cmd/目錄,可以過
2. 改用lotus-box 也是個選擇

**Comment by TippyFlitsUK** [comment]
Hi @seantw1974 

Can you please clarify "sector-checker"? Is this the `lotus proving check` feature or are you referring to a third party package?

Many thanks!

**Reply by seantw1974** [reply]
It's third party package like "lotus-box"
use "lotus-box" can replace it

**Comment by TippyFlitsUK** [comment, marked as answer]
Many thanks for your response @seantw1974 

The upcoming [PoSt Worker](https://github.com/filecoin-project/lotus/pull/7971) enhancement will provide a way to simulate a full Window PoSt as a native Lotus feature!

It is currently in final testing so watch this space for an announcement soon!!
